framework/tst/dslabs/framework/testing/newviz/SingleNodePanel.java

```diff
@@ -99,13 +101,19 @@
         // XXX: why does this need w 100%, h 100%? Shouldn't grow handle it?
         add(mainSplitPane, "grow, h 100%");
 
+        boolean searchDone = settings != null &&
+                             (settings.invariantViolated(s.state()) ||
```

Comment:
I think not allowing expansion past pruned states is fine, but I don't like not allowing exploration past invariant violations or goals.

Reply:
Fixed in https://github.com/emichael/dslabs/pull/29/commits/c0f1eba7cc6966e25eeefa893dd43ac666d0c7a8.